src/simulation/test/LoadGeneratorTests.cpp

```diff
@@ -90,6 +90,166 @@ TEST_CASE("generate load with unique accounts", "[loadgen]")
     }
 }
 
+TEST_CASE("generate soroban load", "[loadgen]")
+{
+    Hash networkID = sha256(getTestConfig().NETWORK_PASSPHRASE);
+    Simulation::pointer simulation =
+        Topologies::pair(Simulation::OVER_LOOPBACK, networkID, [](int i) {
+            auto cfg = getTestConfig(i);
+            cfg.TESTING_UPGRADE_MAX_TX_SET_SIZE = 5000;
+            return cfg;
+        });
+
+    simulation->startAllNodes();
+    simulation->crankUntil(
+        [&]() { return simulation->haveAllExternalized(3, 1); },
+        2 * Herder::EXP_LEDGER_TIMESPAN_SECONDS, false);
+
+    auto nodes = simulation->getNodes();
+    for (auto node : nodes)
+    {
+        overrideSorobanNetworkConfigForTest(*node);
+        modifySorobanNetworkConfig(*node, [&](SorobanNetworkConfig& cfg) {
+            // Entries should never expire
```

Comment:
Yeah I think I can lower these numbers, I just picked a number higher than I thought we would ever simulate to make sure my hard coded fees were sufficient. I think we still need to up the minimum lifetime. We bump the contract data in the contract, but not the instance or Wasm, so those could still expire. I need to redo the loadgen contract anyway, so I think I'm going to remove the rent bump and just have a high minimum lifetime for loadgen so we don't have to worry about restoration and rent bumps. I can also remove the memory override. I think we still need the ledger and write related settings though just to make sure that no otherwise valid TXs get dropped during the setup phase.

Reply:
Yeah, it's fine to override whatever we need, I just wouldn't change more settings than necessary.